commons/build.gradle

```diff
@@ -14,10 +14,15 @@
  * limitations under the License.
  */
 
+ext {
+    zstdVersion = "1.5.4-2"
+}
+
 dependencies {
     // The client of a library should provide a runtime dependency. For now taking the version from kafka-storage-api
     compileOnly 'com.fasterxml.jackson.core:jackson-databind:2.14.2'
     // The client of a library should provide a runtime dependency.
     // For now taking the version from kafka-clients
-    compileOnly "com.github.luben:zstd-jni:1.5.4-2"
+    compileOnly group: "com.github.luben", name: "zstd-jni", version: zstdVersion
+    testImplementation group: "com.github.luben", name: "zstd-jni", version: zstdVersion
```

Comment:
Oh, I missed this in another and merged already but should we preserve the style as it was? e.g.
`testImplementation "com.github.luben:zstd-jni:$zstdVersion"

Reply:
Yeah, let's use one. Updated.